### PR TITLE
analog: add device parameter - battery depassivation config

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: docs/analog/parameter-types.md [hunk 1]
```diff
@@ -736,3 +737,56 @@ hardware type - `0x06` hardware version - `0x0a`
...
+
+## Battery depassivation config
+
+Parameter is used to set thresholds for start/stop device battery depassivation.
+Applies to all types of modules with battery power.
+With the introduction of this parameter, the following listed parameters cease to function.
+
+* [Battery depassivation info](#battery-depassivation-info)
+* [Battery minimal load time](#battery-minimal-load-time)
+
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:750]:
```suggestion
```

File: docs/analog/parameter-types.md [hunk 2]
```diff
@@ -736,3 +737,56 @@ hardware type - `0x06` hardware version - `0x0a`
...
+
+The table displays the version from which this change took effect.
+
+| Device  |        | HARD_TYPE | HARD_VERSION | SOFT_VERSION |
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:754]: what is the second column title?

File: docs/analog/parameter-types.md [hunk 3]
```diff
@@ -736,3 +737,56 @@ hardware type - `0x06` hardware version - `0x0a`
...
+|---------|--------|-----------|--------------|--------------|
+| NOVATOR | SX1276 | 4         | 3            | 0x58         |
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:756]:
```suggestion
| `NOVATOR` | `SX1276` | `4`         | `3`            | `0x58`         |
```

File: docs/analog/parameter-types.md [hunk 4]
```diff
@@ -736,3 +737,56 @@ hardware type - `0x06` hardware version - `0x0a`
...
+| GAZZWLE | WLE    | 12        | 1,5          | 0x70         |
+| WATER   | WLE    | 13        | 2            | 0x09         |
+
+[Hardware types](./basics.md#hardware-types)
+
+### Format
+
+| Size | Type   | Field                                                     |
+|------|--------|-----------------------------------------------------------|
+| 1    | `byte` | parameter type = `33`                                     |
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:768]:
```suggestion
| `1`    | `byte` | parameter type = `33`                                     |
```

File: docs/analog/parameter-types.md [hunk 5]
```diff
@@ -736,3 +737,56 @@ hardware type - `0x06` hardware version - `0x0a`
...
+| 2    | `byte` | [resistance start threshold](#resistance-start-threshold) |
+| 2    | `byte` | [resistance stop threshold](#resistance-stop-threshold)   |
+
+#### **resistance start threshold**
+
+Represents the value of the internal resistance of the battery (in mΩ) upon exceeding which the depassivation process will be initiated. For WLE modules, this value is set by default to 35,000 mΩ, and for modules using the SX1276 transceiver, this value is set to 16,000 mΩ.
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:774]:
```suggestion
Represents the value of the internal resistance of the battery (in `mΩ`) upon exceeding which the depassivation process will be initiated. For WLE modules, this value is set by default to `35000` `mΩ`, and for modules using the `SX1276` transceiver, this value is set to `16000` `mΩ`.
```
?

File: docs/analog/parameter-types.md [hunk 6]
```diff
@@ -736,3 +737,56 @@ hardware type - `0x06` hardware version - `0x0a`
...
+* [Battery minimal load time](#battery-minimal-load-time)
+
+
+The table displays the version from which this change took effect.
+
+| Device  |        | HARD_TYPE | HARD_VERSION | SOFT_VERSION |
+|---------|--------|-----------|--------------|--------------|
+| NOVATOR | SX1276 | 4         | 3            | 0x58         |
+| NOVATOR | WLE    | 4         | 4            | 0x70         |
+| IMP4EU  | WLE    | 6         | 16,17,18,19  | 0x74         |
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:758]: I would suggest add spaces after commas

File: docs/analog/parameter-types.md (hunk 2 above)
Comment (DeadByElpy) [reply docs/analog/parameter-types.md]: fixed

File: docs/analog/parameter-types.md (hunk 3 above)
Comment (DeadByElpy) [reply docs/analog/parameter-types.md]: fixed

File: docs/analog/parameter-types.md (hunk 4 above)
Comment (DeadByElpy) [reply docs/analog/parameter-types.md]: fixed

File: docs/analog/parameter-types.md (hunk 5 above)
Comment (DeadByElpy) [reply docs/analog/parameter-types.md]: fixed

File: docs/analog/parameter-types.md (hunk 6 above)
Comment (DeadByElpy) [reply docs/analog/parameter-types.md]: fixed

File: docs/analog/parameter-types.md [hunk 7]
```diff
@@ -736,3 +737,55 @@ hardware type - `0x06` hardware version - `0x0a`
...
+| `2`    | `byte` | [resistance start threshold](#resistance-start-threshold) |
+| `2`    | `byte` | [resistance stop threshold](#resistance-stop-threshold)   |
+
+#### **resistance start threshold**
+
+Represents the value of the internal resistance of the battery (in `mΩ`) upon exceeding which the depassivation process will be initiated. For WLE modules, this value is set by default to `35000` `mΩ`, and for modules using the `SX1276` transceiver, this value is set to `16000` `mΩ`.
+
+#### **resistance stop threshold**
+
+Represents the value of the internal resistance of the battery (in `mΩ`). If the internal resistance of the battery falls below this threshold, the depassivation process will stop. For WLE modules, this value is set by default to `25000` `mΩ`, and for modules using the SX1276 transceiver, this value is set to `10350` `mΩ`.
```
[the remark this answers is not in the document]
Comment (DeadByElpy) [reply docs/analog/parameter-types.md]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**